WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -58,6 +58,12 @@ extension UIImage {
             .applyTintColor(.placeholderImage)!
     }
 
+    /// Card Reader Scanning Image
+    ///
+    static var cardReaderScanningImage: UIImage {
```

Comment:
We usually add a unit test to cover images. I have personally been bitten in the past by runtime crashes due to assets being missed or typos in the asset name. 

You can look at the existing tests in IconsTests

Reply:
Thanks! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3775/commits/091b86d65eb8063e61c236a40c9a887d595efac1